integration/it-integration/src/main/java/se/inera/intyg/rehabstod/integration/it/dto/SickLeavesResponseDTO.java

```diff
@@ -20,20 +20,16 @@
 package se.inera.intyg.rehabstod.integration.it.dto;
 
 import java.util.List;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 import se.inera.intyg.infra.sjukfall.dto.SjukfallEnhet;
 
+@Data
+@NoArgsConstructor
 public class SickLeavesResponseDTO {
     private List<SjukfallEnhet> content;
 
     public SickLeavesResponseDTO(List<SjukfallEnhet> content) {
```

Comment:
Since we decided to use @NoArgsConstructor, maybe we can use @AllArgsConstructor to generate this constructor?